var/spack/repos/builtin/packages/tandem/package.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class Tandem(CMakePackage):
+    """Tandem is a scientific software for SEAS modelling and for solving Poisson
+    and linear elasticity problems. It implements the Symmetric Interior Penalty
+    Galerkin (SIPG) method using unstructured simplicial meshes (triangle meshes
+    in 2D, tetrahedral meshes in 3D)."""
+
+    homepage = "https://tandem.readthedocs.io/en/latest/"
+    version("main", git="https://github.com/TEAR-ERC/tandem.git", branch="main", submodules=True)
```

Comment:
Curious.  Why not just set the `git` property since it is unchanged for the two versions?

(See https://spack.readthedocs.io/en/latest/packaging_guide.html#git)

Reply:
Thank you I did not know about that.